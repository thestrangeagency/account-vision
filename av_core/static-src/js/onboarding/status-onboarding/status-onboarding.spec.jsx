import React from 'react';
import { shallow } from 'enzyme';

import StatusOnboarding from './status-onboarding';

jest.mock('../../services/onboarding-api-service');

test('it renders', () => {
  const wrapper = shallow(<StatusOnboarding nextPage="" nextPageAlt="" />);
  expect(wrapper).toMatchSnapshot();
});
