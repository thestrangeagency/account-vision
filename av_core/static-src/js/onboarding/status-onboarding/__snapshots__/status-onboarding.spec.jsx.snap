// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<OnboardingForm
  nextPage={[Function]}
  steps={
    Array [
      Object {
        "inputs": Array [
          Object {
            "inputProps": Object {
              "initial": null,
              "label": "Filing Status",
              "name": "filing_status",
              "required": true,
              "selectChoices": Array [
                Object {
                  "text": "---------",
                  "value": "",
                },
                Object {
                  "text": "Single",
                  "value": "SINGLE",
                },
                Object {
                  "text": "Married, filing jointly",
                  "value": "MARRIED_JOINT",
                },
                Object {
                  "text": "Married, filing separately",
                  "value": "MARRIED_SEPARATE",
                },
                Object {
                  "text": "Head of household",
                  "value": "HEAD",
                },
                Object {
                  "text": "Widow",
                  "value": "WIDOW",
                },
              ],
            },
            "type": "SELECT",
          },
        ],
      },
    ]
  }
  userId={null}
/>
`;
